soundland/src/Components/genre_dd.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+
+class GenreDown extends React.Component {
+    constructor(props) {
+      super(props);
+
+      this.state = {value: 'Rock'};
+      
+          this.handleChange = this.handleChange.bind(this);
+          this.handleSubmit = this.handleSubmit.bind(this);
+        }
+      
+        handleChange(event) {
+          this.setState({value: event.target.value});
```

Comment:
@yardenhochman will this.target.value be retrieved on a drop down options? Do you not need to go after the name attribute?

Reply:
I'm still working on adding gary's file to dev